pkg/cmd/grafana-kiosk/main.go

```diff
@@ -44,6 +46,8 @@ func ProcessArgs(cfg interface{}) Args {
 	f.StringVar(&a.Mode, "kiosk-mode", "full", "Kiosk Display Mode [full|tv|disabled]\nfull = No TOPNAV and No SIDEBAR\ntv = No SIDEBAR\ndisabled = omit option\n")
 	f.StringVar(&a.URL, "URL", "https://play.grafana.org", "URL to Grafana server")
 	f.StringVar(&a.WindowPosition, "window-position", "0,0", "Top Left Position of Kiosk")
+	f.StringVar(&a.WindowPosition, "window-size", "1920,1080", "Size of Kiosk in pixels")
```

Comment:
this variable should be `a.WindowSize` or it will overwrite the position setting and not set the size